enterprise-suite/values.yaml

```diff
@@ -90,6 +90,12 @@ exposeServices: false
 # If exposeServices is set to "NodePort" or "LoadBalancer", Console will be exposed on this port.
 esConsoleExposePort: 30080
 
+# Path to custom default monitors file, must be inside the Console Helm Chart directory.
+defaultMonitorsFile:
+
+# Path to custom static rules file, must be inside the Console Helm Chart directory.
+staticRulesFile:
+
```

Comment:
I think this is too limiting - we shouldn't require users to have to check out the repo and do a local chart install to edit the monitors.

Instead these should refer to ConfigMap names - which will let users override with their own ConfigMap.

I'm also thinking we should install the configmaps as well for the default files - and set them. This gives an example to users how it works (rather than it being implicit via the default argument on the image). So the default value for these should be configmaps we create using the copied over files. This also lets users easily copy those files out of the configmaps for tuning.

Reply:
Updated to use ConfigMaps and to default to configs copied over from `console-api`